packages/lambda-powertools-middleware-correlation-ids/index.d.ts

```diff
@@ -0,0 +1,62 @@
+import middy from "@middy/core";
+import {
+  Context,
+  SQSRecord,
+  Handler,
+  DynamoDBRecord,
+  DynamoDBStreamEvent,
+  Callback,
+  FirehoseTransformationEvent,
+  FirehoseTransformationResult,
+  KinesisStreamEvent,
+} from "aws-lambda";
+import Log from "@dazn/lambda-powertools-logger";
+import CorrelationIds from "@dazn/lambda-powertools-correlation-ids";
+
+export default function <T, R, C extends Context = Context>(params: {
+  sampleDebugLogRate: number;
+}): middy.MiddlewareObject<T, R, C>;
+
+export type ExtractedCorrelationIdAndLogger<L> = {
```

Comment:
```suggestion
export type ExtractedCorrelationIdAndLogger<L = Log> = {
```

Default value like the rest?

Reply:
Done! LGTM 👀  😄